hero/challenger dominiel.json

```
@@ -0,0 +1,198 @@
+{
+    "name": "Challenger Dominiel",
+    "rarity": 4,
+    "classType": "mage",
+    "element": "dark",
+    "zodiac": "scorpio",
+    "specialtyChangeName": "",
+    "selfSkillBarName": "",
+    "background": [
+        "Fiercely proud and unreserved, Challenger Dominiel has little morality or sense of empathy. While constantly causing trouble, she refuses to admit her faults."
+    ],
+    "relations": [
+        { "hero": "dingo", "relationType": "rival" },
+        { "hero": "zerato", "relationType": "rival" }
+    ],
+    "stats": {
+        "base": {
+            "cp": 1274,
+            "atk": 90,
+            "hp": 230,
+            "spd": 96,
+            "def": 74,
+            "chc": 15,
+            "chd": 150,
+            "eff": 0,
+            "efr": 0,
+            "dac": 5
+        },
+        "max": {
+            "cp": 15594,
+            "atk": 1187,
+            "hp": 4491,
+            "spd": 96,
+            "def": 627,
+            "chc": 24,
+            "chd": 156,
+            "eff": 0,
+            "efr": 0,
+            "dac": 5
+        }
+    },
+    "skills": [
+        {
+            "isPassive": false,
+            "soulBurn": 20,
+            "selfSkillBarValue": 0,
+            "soulBurnEffect": "Increases damage dealt.",
+            "awakenUpgrade": false,
+            "cooldown": 0,
+            "name": "Electrocute",
+            "soulAcquire": 1,
+            "description": "Attacks with electricity. A critical hit will increase damage dealt.",
+            "enhancement": [
+                { "description": "+5% damage dealt", "resources": [{ "item": "gold", "qty": 4000 } , { "item": "molagora", "qty": 1 }] },
+                { "description": "+5% effect chance", "resources": [{ "item": "gold", "qty": 8000 } , { "item": "molagora", "qty": 2}] },
+                { "description": "+10% damage dealt", "resources": [{ "item": "gold", "qty": 17000 } , { "item": "sharp-spearhead", "qty": 1} , { "item": "molagora", "qty": 3 }] },
+                { "description": "+10% effect chance", "resources": [{ "item": "gold", "qty": 35000 } , { "item": "sharp-spearhead", "qty": 3} , { "item": "molagorago", "qty": 1 }] },
+                { "description": "+15% damage dealt", "resources": [{ "item": "gold", "qty": 100000 } , { "item": "black-powder-curse", "qty": 2} , { "item": "molagorago", "qty": 2 }] }
+            ],
+            "buffs": [],
+            "debuffs": []
+        },
+        {
+            "isPassive": true,
+            "soulBurn": 0,
+            "soulBurnEffect": "",
+            "awakenUpgrade": true,
+            "cooldown": 0,
+            "name": "Electric Charge",
+            "soulAcquire": 0,
+            "description": "When an ally makes a critical hit, the caster's next attack deals stacked additional damage proportional to the number of critical hits and their combat readiness increases by 10% per critical hit.",
```

Missing caps on Combat Readiness.
```suggestion
            "description": "When an ally makes a critical hit, the caster's next attack deals stacked additional damage proportional to the number of critical hits and their Combat Readiness increases by 10% per critical hit.",
```